sanic/sanic.py

```diff
@@ -247,7 +243,7 @@ def converted_response_type(self, response):
     def run(self, host="127.0.0.1", port=8000, debug=False, before_start=None,
             after_start=None, before_stop=None, after_stop=None, sock=None,
             workers=1, loop=None, protocol=HttpProtocol, backlog=100,
-            stop_event=None):
+            stop_event=None, logger=None):
```

Comment:
@awiddersheim oh I thought you were talking about stop_event :( derp, I forgot to remove the logger boolean here and you're right that was not needed